### PR TITLE
added value column tables to Accumulator

### LINKED ISSUE #27: Use a value table for the columns and aggregators in propagator.Accumulator
same as #17, but for the Accumulator

### PR DESCRIPTION
**Pull request by gcfang**
closes #27

### CODE REVIEW

**Comment by gcfang** [comment]
```
C:\Users\cfang\Documents\GitHub\python-propagator [accu_value_table]> nosetests --with-coverage --cover-package=propagat
or
............................................................................F...........................................
..............................................................
======================================================================
FAIL: propagator.tests.test_utils.Test_Extension.test_unlicensed_extension
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\cfang\AppData\Local\Continuum\Anaconda2_32\lib\site-packages\nose\case.py", line 197, in runTest
    self.test(*self.arg)
  File "C:\Users\cfang\AppData\Local\Continuum\Anaconda2_32\lib\site-packages\nose\tools\nontrivial.py", line 67, in new
func
    raise AssertionError(message)
AssertionError: test_unlicensed_extension() did not raise RuntimeError

Name                     Stmts   Miss  Cover   Missing
------------------------------------------------------
propagator.py                3      0   100%
propagator\analysis.py     163      1    99%   771
propagator\base_tbx.py      79      1    99%   186
propagator\toolbox.py      187     17    91%   410, 414, 416-417, 578-579, 582-596
propagator\utils.py        261      5    98%   199-200, 318, 689, 1469
propagator\validate.py      29      1    97%   143
------------------------------------------------------
TOTAL                      722     25    97%
----------------------------------------------------------------------
Ran 182 tests in 16.851s
```

**Comment by phobson** [comment]
I'll take a look at this in the morning.

File: propagator/tests/test_toolbox.py [hunk 1]
```diff
@@ -139,13 +139,22 @@ def test_filtered(self):
...
             ds_col='Dwn_Catch_',
-            area_col='Area',
-            imp_col='Imp',
+            value_columns = [
+            ('medDry_M', 'maximum','n/a'),
+            ('p10Dry_N', 'First','area'),
+            ('Wet_B', 'WeIghtED_AveragE','imp_area'),
+            ('aveWet_M', 'minimum','imp_area'),
+            ('maxWet_N', 'average','n/a'),
+            ('Area', 'sum','n/a'),
```
Comment (phobson) [inline propagator/tests/test_toolbox.py]: indent this block 1 level

File: propagator/toolbox.py [hunk 2]
```diff
@@ -525,39 +547,88 @@ def _params_as_list(self):
...
+
+            if params['subcatchments'].value:
+                fields = analysis._get_wq_fields(sc, prefix)
+                fields.append('n/a')
+                self._set_filter_list(vc.filters[0], fields)
+                self._set_filter_list(vc.filters[1], list(analysis.AGG_METHOD_DICT.keys()))
+                self._set_filter_list(vc.filters[2], fields)
+            self._update_value_table_with_default(vc, ['sum','n/a'])
+
+    # @property
```
Comment (phobson) [inline propagator/toolbox.py]: remove unused code

**Comment by phobson** [comment]
This is looking good. You'll need to write another test to two that fills in the coverage gap created by these changes. You can use the coverage report as a guide to see which code paths in toolbox.py aren't getting hit.

Also, we'll want to rebase/squash some of the WIP commits. I'll help you do that when the time comes.

**Comment by gcfang** [comment]
```
C:\Users\cfang\Documents\GitHub\python-propagator [accu_value_table]> nosetests --with-coverage --cover-package=propagat
or
.............................................................................F..........................................
...............................................................
======================================================================
FAIL: propagator.tests.test_utils.Test_Extension.test_unlicensed_extension
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\cfang\AppData\Local\Continuum\Anaconda2_32\lib\site-packages\nose\case.py", line 197, in runTest
    self.test(*self.arg)
  File "C:\Users\cfang\AppData\Local\Continuum\Anaconda2_32\lib\site-packages\nose\tools\nontrivial.py", line 67, in new
func
    raise AssertionError(message)
AssertionError: test_unlicensed_extension() did not raise RuntimeError

Name                     Stmts   Miss  Cover   Missing
------------------------------------------------------
propagator.py                3      0   100%
propagator\analysis.py     163      1    99%   771
propagator\base_tbx.py      79      1    99%   186
propagator\toolbox.py      187      4    98%   410, 414, 416-417
propagator\utils.py        261      5    98%   199-200, 318, 689, 1469
propagator\validate.py      29      0   100%
------------------------------------------------------
TOTAL                      722     11    98%
----------------------------------------------------------------------
Ran 183 tests in 31.392s
```

**Pull request merged**